src/fluentexpr.prw

```diff
@@ -18,10 +18,20 @@ Static Function Format( cString, aValues )
     Local cResult := cString
     Local nIndex
     For nIndex := 1 To Len( aValues )
-        cResult := StrTran( cResult, '{' + AllTrim( Str( nIndex ) ) + '}', cValToChar( aValues[ nIndex ] ) )
+        cResult := StrTran( cResult, '{' + AllTrim( Str( nIndex ) ) + '}', ToString( aValues[ nIndex ] ) )
     Next
     Return cResult
 
+Static Function ToString( xValue )
+    If ValType( xValue ) == "A"
+        Return '{ ' + ArrTokStr(xValue, ', ') + ' }'
+    ElseIf ValType( xValue ) == "B"
+        Return GetCBSource( xValue )
+    ElseIf ValType( xValue ) == "O"
+        Return ArrTokStr( ClassMethArr( xValue, .T. ), ' | ' )
```

Comment:
Objects are compared both by methods and data properties.
A good way to show it would be having two sections (methods and properties). `ClassDataArr` also should be used, and the parameters received by the method are also available on `ClassMethArr`.

Reply:
We could implement this in another PR, right?